app/src/main/res/values/strings.xml

```diff
@@ -821,21 +822,24 @@ Signature: %5$s</string>
     <string name="released">Released</string>
 
     <string name="including_format_string">including %s</string>
-    <string name="new_items_format_string">%d new items</string>
+    <plurals name="new_items_format_string">
+        <item quantity="one">%d new item</item>
```

Comment:
nit: If this is the only option, I always feel like this could be more obvious to future maintainers
```suggestion
        <item quantity="one">1 new item</item>
```